src/uct/cuda/cuda_ipc/cuda_ipc_iface.h

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2018.  ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+#ifndef UCT_CUDA_IPC_IFACE_H
+#define UCT_CUDA_IPC_IFACE_H
+
+#include <uct/base/uct_iface.h>
+#include <ucs/arch/cpu.h>
+#include <cuda_runtime.h>
+#include <cuda.h>
+
+
+#define UCT_CUDA_IPC_TL_NAME    "cuda_ipc"
+#define UCT_CUDA_DEV_NAME   "cudaipc0"
+
+
+#define UCT_CUDA_FUNC(_func)  ({                        \
```

Comment:
copy/paste from codacopy, move to common header

Reply:
this macro is not used in this PR.  removed it,  will move it to common header when it is used.